adapters/handlers/rest/configure_weaviate.go

```diff
@@ -23,7 +23,7 @@ import (
 
 var connectorOptionGroup *swag.CommandLineOptionsGroup
 
-// configureAPI -> see configure_api.go
+// ConfigureAPI -> see configure_api.go
```

Comment:
here also, can we change it back to `configureAPI` as we are still using this method for initing Weaviate